packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -58,6 +62,71 @@ - (UIViewController *)viewControllerWithWindow:(UIWindow *)window {
   return topController;
 }
 
+- (void)pickImage:(bool)single {
+  if (@available(iOS 14, *)) {
+    PHPickerConfiguration *config = [[PHPickerConfiguration alloc] init];
+    self->_single = single;
+    if (!single) config.selectionLimit = 0;
+    config.filter = [PHPickerFilter imagesFilter];
+
+    PHPickerViewController *pickerViewController =
+        [[PHPickerViewController alloc] initWithConfiguration:config];
+    pickerViewController.delegate = self;
+    [[self viewControllerWithWindow:nil] presentViewController:pickerViewController
+                                                      animated:YES
+                                                    completion:nil];
+  }
+}
+
+- (void)picker:(PHPickerViewController *)picker
+    didFinishPicking:(NSArray<PHPickerResult *> *)results API_AVAILABLE(ios(14)) {
+  [picker dismissViewControllerAnimated:YES completion:nil];
+  NSMutableArray *pathList = [NSMutableArray new];
+  for (PHPickerResult *result in results) {
+    [result.itemProvider
+        loadDataRepresentationForTypeIdentifier:@"public.image"
+                              completionHandler:^(NSData *_Nullable data,
+                                                  NSError *_Nullable error) {
+                                NSLog(@"data: %lu", (unsigned long)data.length);
+                                if (data != nil) {
+                                  NSArray *paths = NSSearchPathForDirectoriesInDomains(
+                                      NSDocumentDirectory, NSUserDomainMask, YES);
+                                  NSString *documentsDirectory = [paths objectAtIndex:0];
+
+                                  NSString *filename =
+                                      [NSString stringWithFormat:@"%@.png",
+                                                                 result.itemProvider.suggestedName];
+                                  NSString *path =
+                                      [documentsDirectory stringByAppendingPathComponent:filename];
+                                  NSNumber *maxWidth = [self->_arguments objectForKey:@"maxWidth"];
+                                  NSNumber *maxHeight =
+                                      [self->_arguments objectForKey:@"maxHeight"];
+                                  NSNumber *imageQuality =
+                                      [self->_arguments objectForKey:@"imageQuality"];
+                                  if (maxWidth != (id)[NSNull null] ||
+                                      maxHeight != (id)[NSNull null]) {
+                                    path = [FLTImagePickerPhotoAssetUtil
+                                        saveImageWithOriginalImageData:data
```

Comment:
The resizing of images seems to give an exception. We're currently working on a PHPicker implementation that fixes a different iOS 14 issue, we will rebase on that once it is finished and merged